doc/nrf/links.txt

```diff
@@ -262,6 +262,8 @@
 
 .. _`Matter weather station application from the v2.1.1`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/2.1.1/nrf/applications/matter_weather_station/README.html
 
+.. _`upgradable second-stage bootloader`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_bootloader.html#second-stage-upgradable-bootloader
```

Comment:
```suggestion
```
As in earlier comment, you can dump this and use the "tag" in the target doc instead.